sharepoint/sharepoint-ps/sharepoint-online/Start-SPORestrictedAccessForSitesInsights.md

```diff
@@ -0,0 +1,119 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/start-sporestrictedaccessforsitesinsights
+applicable: SharePoint Online
+title: Start-SPORestrictedAccessForSitesInsights
+schema: 2.0.0
+author: itambakhe
+ms.author: itambakhe
+ms.reviewer:
+manager:
+---
+ 
+# Start-SPORestrictedAccessForSitesInsights
+
+## SYNOPSIS
+
+This cmdlet enables administrator to trigger the build of a new Restricted Access Control insights report for the data from last 28 days.
+
+## SYNTAX
+
+```powershell
+Start-SPORestrictedAccessForSitesInsights -RACProtectedSites [-Force <SwitchParameter>] 
+```
+
+```powershell
+Start-SPORestrictedAccessForSitesInsights -ActionsBlockedByPolicy [-Force <SwitchParameter>] 
+```
+
+## DESCRIPTION
+
+After this cmdlet is executed, the Restricted Access Control insights report generation request for the requested sub type gets queued in the pipeline and the below metadata is displayed with the following properties:
```

Comment:
>After this cmdlet is executed, the Restricted Access Control insights report generation request for the requested sub type gets queued in the pipeline and the below metadata is displayed with the following properties [](http://example.com/codeflow?start=0&length=216)

This sentence could be simplified. e.g., you could remove stuff like "gets queued in the pipeline" to just "the request is initiated" etc